InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/auto/AutoDrive.java

```diff
@@ -68,6 +68,7 @@ public void execute() {
     public boolean isFinished() {
         // whether the RAMSETE command trajectory has been executed
         boolean trajDone = super.isFinished();
+        trajDone = false;
```

Comment:
Can we drop this? It seems like a change we don't want in the mainline.

Reply:
Yep, I thought this got removed when I merged in from the master repo.